bundles/EcommerceFrameworkBundle/Model/AbstractOrderItem.php

```diff
@@ -123,43 +124,43 @@ public function setTotalPrice($totalPrice)
     }
 
     /**
-     * Should return a float
+     * @throws UnsupportedException
      *
-     * @return void
+     * @return float
      */
     public function getTotalNetPrice()
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('getTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('getTotalNetPrice not implemented for ' . get_class($this));
     }
 
     /**
+     * @throws UnsupportedException
+     *
      * @param float $totalNetPrice
      */
     public function setTotalNetPrice($totalNetPrice)
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('setTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('setTotalNetPrice not implemented for ' . get_class($this));
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7553